src/tasks/components/NewTaskScheduler/TaskScheduler.tsx

```diff
@@ -0,0 +1,181 @@
+import React, {FC, ChangeEvent, useCallback, useEffect, useState} from 'react'
+
+// Components
+import {
+  Button,
+  ButtonShape,
+  ButtonType,
+  Columns,
+  ComponentColor,
+  Form,
+  Grid,
+  IconFont,
+  Input,
+  Page,
+  SelectGroup,
+} from '@influxdata/clockface'
+import TaskSchedulerFormField from 'src/tasks/components/NewTaskScheduler/TaskSchedulerFormField'
+import ScriptSelector from 'src/tasks/components/NewTaskScheduler/ScriptSelector'
+
+// Utils
+import {debouncer} from 'src/dataExplorer/shared/utils'
```

Comment:
I think we should use lodash's `debounce`, rather than the home rolled one, since it's maintained and more familiar to most people.